### ISSUE TITLE
Translate injuries

### ISSUE THREAD

**Issue by sebastiaanspeck** [sebastiaanspeck/sportmonks#4]
**Is your feature request related to a problem? Please describe.**
The injuries that are included with the sidelined-players are still in English.

**Describe the solution you'd like**
I would like to have an injuries.php with the translations of the injuries. If someone is willing to pick this, I will add all the injuries (still have to ask the API-provider for a list of all possible injuries)

**Additional context**
More context will be added when someone is picking this issue.

**Comment by DBX12** [comment]
I can do the German translation. Should I provide them in a `injuries-de.php` file or in an other way?

**Comment by sebastiaanspeck** [comment]
Great! And name it `injuries.php` ;) Good luck!

**Comment by sebastiaanspeck** [comment]
@tribakzero Could you do this for Spanish?

**Comment by sebastiaanspeck** [comment]
@davidmohamedfr Could you do this for French?

**Comment by sebastiaanspeck** [comment]
@nipeharefa Could you do this for Indonesian? **(Pay attention you change the injuries.php in the correct folder (lang/id)**

**Comment by tribakzero** [comment]
Sorry, reviewed the list but I'm not aware of most of those injuries names in spanish.

**Comment by sebastiaanspeck** [comment]
You can make a start and I will find someone who can finish it.

**Comment by davidmohamedfr** [comment]
> @davidmohamedfr Could you do this for French?

@sebastiaanspeck Yes, but I've no medical background so it will be an approximation.

**Comment by sebastiaanspeck** [comment]
> > @davidmohamedfr Could you do this for French?
> 
> @sebastiaanspeck Yes, but I've no medical background so it will be an approximation.

Thats okay as starting point.

**Comment by nipeharefa** [comment]
@sebastiaanspeck 
Okay

**Comment by AangPer** [comment]
I've updated almost everything in Spanish, it seems silly but "knock" I don't know if it refers just to a knock, hit, stroke or has another meaning in English

**Comment by sebastiaanspeck** [comment]
> @sebastiaanspeck
> Okay

Any update?

**Comment by sebastiaanspeck** [comment]
> I've updated almost everything in Spanish, it seems silly but "knock" I don't know if it refers just to a knock, hit, stroke or has another meaning in English

https://www.reddit.com/r/chelseafc/comments/7c6yzm/what_is_a_knock_injury/
